examples/dataset_group.py

```diff
@@ -1,4 +1,7 @@
 from servicex import ServiceXClient, RucioDatasetIdentifier, ResultFormat, DatasetGroup
+from rich.console import Console
```

Comment:
ALl this looks great - other than we aren't using `logging` - but how did this work before - I see "Console" used below and it doesn't show up in this diff! :-)

Reply:
Oh - I see - the include was down below. :-)

Reply:
yeah - just a flake8 issue about my imports lower in the file